src/library_syscall.js

```diff
@@ -720,7 +720,12 @@ var SyscallsLibrary = {
 
         return bytesRead;
       }
-      default: abort('unsupported socketcall syscall ' + call);
+      default: {
+#if SYSCALL_DEBUG
+        err('    (socketcall: ' + call + ')');
```

Comment:
why drop the "unsupported" part?

Reply:
In this same function the other err() functions used this style. I can revert back the format if you'd like.